shunting-yard-binary/pom.xml

```diff
@@ -2,9 +2,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>com.hotels</groupId>
+    <groupId>com.expediagroup</groupId>
     <artifactId>shunting-yard</artifactId>
-    <version>2.0.2-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>shunting-yard-binary</artifactId>
```

Comment:
The shunting yard dependencies below also need to be updated to reflect the new group id.

Reply:
done.

Reply:
done.